Fido/gradle.properties

```diff
@@ -10,6 +10,8 @@
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
 # Default value: -Xmx10248m -XX:MaxPermSize=256m
+android.enableJetifier=true
```

Comment:
Line 15 should be directly under the comment above. Let's move lines 13 and 14 to the bottom of the file.

Reply:
Thanks. Changed the order of lines.